components/automate-chef-io/content/docs/configuration.md

```diff
@@ -290,7 +290,7 @@ which should be carefully reviewed and considered.
 
 For the purposes of an example, a system with 32GB of memory can have its Elasticsearch heap
 size set to `16g`; to do so, one would first create a TOML file that contains the partial
-configuration below, and then issue a `chef-automate config patch </path/to/your-file.toml>` to
+configuration below, uncomment and change settings as neededand, then issue a `chef-automate config patch </path/to/your-file.toml>` to
```

Comment:
```suggestion
configuration below, uncomment and change settings as needed and, then run `chef-automate config patch </path/to/your-file.toml>` to
```

Reply:
@susanev Ended up with a merge conflict, but here's the new fix.

Reply:
@kgarmoe Looks like this was merged already but there are still changes to be made. It was merged without your correction to `neededand`, and also, that entire statement doesn't make sense here because of the switch to a presentation of an example as described in the paragraph (this makes more sense to users). Nothing is commented out in the configuration snippet, so `uncomment and change settings as needed and` should just be removed.

Reply:
ill open a follow up in a few minutes